airbyte-integrations/connectors/destination-iceberg-v2/src/test-integration-legacy/kotlin/io/airbyte/integrations/destination/iceberg_v2/IcebergV2SpecTest.kt

```diff
@@ -0,0 +1,5 @@
+package io.airbyte.integrations.destination.iceberg_v2
+
+import io.airbyte.cdk.load.spec.SpecTest
+
+class IcebergV2SpecTest : SpecTest()
```

Comment:
fyi: there's probably some way to get JUnit to run the base `SpecTest` directly, but I couldn't get it working, and writing this class is easy enough that I didn't particularly care :P